server/datastore/mysql/migrations/tables/20240129162819_AddPrefixToWindowsUpdatesProfiles_test.go

```diff
@@ -0,0 +1,56 @@
+package tables
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUp_20240129162819(t *testing.T) {
+	db := applyUpToPrev(t)
+
+	prof1UUID := "w" + uuid.NewString()
+	updateProfUUID := uuid.NewString()
+	setupStmt := `
+		INSERT INTO mdm_windows_configuration_profiles VALUES
+			(0,'prof1','<Replace></Replace>','2023-11-03 20:32:32','2023-11-03 20:32:32', '%s'),
+			(0,'updateProf','<Replace></Replace>','2023-11-03 21:32:32','2023-11-03 21:32:32', '%s');
+		INSERT INTO host_mdm_windows_profiles (host_uuid, command_uuid, profile_uuid) VALUES
+			('1', '1', '%s'),
+			('2', '2', '%s');
+	`
+
+	_, err := db.Exec(fmt.Sprintf(setupStmt, prof1UUID, updateProfUUID, prof1UUID, updateProfUUID))
+	require.NoError(t, err)
+
+	// Apply current migration.
+	applyNext(t, db)
+
+	stmt := `SELECT profile_uuid FROM mdm_windows_configuration_profiles;`
+	rows, err := db.Query(stmt)
```

Comment:
Note that you can use `sqlx.Select` to query into a slice so you don't have to do the rows looping and scanning manually.